webapp/javascript/components/TimelineChart/AnnotationMark/index.tsx

```diff
@@ -0,0 +1,65 @@
+import Color from 'color';
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import AnnotationInfo from '@webapp/pages/continuous/contextMenu/AnnotationInfo';
+import useTimeZone from '@webapp/hooks/timeZone.hook';
+import { faCommentDots } from '@fortawesome/free-regular-svg-icons/faCommentDots';
+
+import styles from './styles.module.scss';
+
+interface IAnnotationMarkProps {
+  type: 'message';
+  color: Color;
```

Comment:
Pretty sure color can be inferred from the `type` (which we currently only have `message`). Each type is gonna be an item/bg color, which is static. We won't allow people to mix and match these. At least not in the foreseeable future.

Reply:
can you please explain more what you want to see here